### DISCUSSION TITLE
multiple languages

### DISCUSSION THREAD

**Discussion by DewittBlack** [torrust/torrust-index-gui#293] [Feature requests]
I hope to add a feature for switching between multiple languages, or provide text for other users to help with translations together.

**Comment by josecelano** [comment]
> I hope to add a feature for switching between multiple languages, or provide text for other users to help with translations together.

Hi @DewittBlack, that would be great! We've been [discussing localization](https://github.com/torrust/.github/issues/12).

Do you have any particular idea about how to implement it?

I think it would be good to start from this repo. Maybe only the application labels (excluding content generated by the user and server messages for now), but we should consider the whole solution as @da2ce7 proposed [here](https://github.com/torrust/.github/issues/12#issuecomment-1691663902).

I think a good step could be extracting hardcoded labels in this GUI into a separate file. But no idea how this could be done in a good way. I would start by reading the official docs https://v8.i18n.nuxtjs.org/
